Analytics/Controllers/AnalyticsTreeController.cs

```diff
@@ -96,8 +98,13 @@ protected override Umbraco.Web.Models.Trees.TreeNodeCollection GetTreeNodes(stri
         /// <returns></returns>
         protected override Umbraco.Web.Models.Trees.MenuItemCollection GetMenuForNode(string id, System.Net.Http.Formatting.FormDataCollection queryStrings)
         {
-            return null;
+            var menu = new MenuItemCollection();
+            if (id == Constants.System.Root.ToInvariantString()) 
+            {
+                // root actions
+                menu.Items.Add<RefreshNode, ActionRefresh>(ui.Text("actions", ActionRefresh.Instance.Alias), false);
+            }
+            return menu;
```

Comment:
@warrenbuckley yep, however it would be nice if the context menu only is available when there is any actions, but it should be something fixed in the core: http://issues.umbraco.org/issue/U4-5161